src/explore-education-statistics-common/src/modules/release/components/OfficialStatisticsSection.tsx

```diff
@@ -33,6 +33,26 @@ export default function OfficialStatisticsSection({
         </a>
         .
       </p>
+      <p>
+        Our statistical practice is regulated by the Office for Statistics
+        Regulation (OSR).
+      </p>
+      <p>
+        OSR sets the standards of trustworthiness, quality and value in the{' '}
+        <a href="https://code.statisticsauthority.gov.uk/the-code/">
+          Code of Practice for Statistics
+        </a>{' '}
+        that all producers of official statistics should adhere to.
+      </p>
+      <p>
+        You are welcome to contact us directly with any comments about how we
+        meet these standards. Alternatively, you can contact OSR by emailing{' '}
+        <a href="mailto:regulation@statistics.gov.uk">
+          regulation@statistics.gov.uk
+        </a>{' '}
+        or via the{' '}
+        <a href="https://osr.statisticsauthority.gov.uk/">OSR website</a>.
```

Comment:
Should this be using a `target="_blank"`?

Reply:
Actually you could use the existing Link component.  I think it'll give you this for free.